src/lib/routes/frontsite/Frontsite.js

```diff
@@ -77,6 +78,7 @@ export default class FrontSite extends Component {
 
 FrontSite.propTypes = {
   customStaticPages: PropTypes.func,
+  config: PropTypes.object.isRequired,
```

Comment:
Same as before why required? Sorry if I am busting nerves..

Reply:
IMO without the config the app will not work, that is why it should be mandatory

Reply:
Perfect! What is missing? For example if a `title` is required this config should be a shape with a title required.

Reply:
Let's go philosophical on this one, what should be required for an App to work?